File: OdontoSysFrontEnd/OdontoSysModel/Comprobante.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace OdontoSysModel
{
    public class Comprobante
    {
        public int IdComprobante { get; set; }
        public DateTime FechaEmision { get; set; }
        public DateTime HoraEmision { get; set; }
        public double Total { get; set; }
        public MetodoPago MetodoDePago { get; set; }

        public Comprobante()
        {
            this.MetodoDePago = new MetodoPago();
        }

        public override string ToString()
        {
            return $"Comprobante{{IdComprobante={IdComprobante}, FechaEmision={FechaEmision}, HoraEmision={HoraEmision}, Total={Total}, MetodoDePago={MetodoDePago}}}";
        }
    }
}

File: BackEnd/OdontoSysModel/src/main/java/pe/pucp/edu/odontosys/services/model/Comprobante.java
package pe.pucp.edu.odontosys.services.model;

import pe.pucp.edu.odontosys.infrastructure.model.MetodoPago;

public class Comprobante {
    
    private Integer idComprobante;
    private String fechaEmision;
    private String horaEmision;
    private double total;
    private MetodoPago metodoDePago;

    public Comprobante() {
        this.metodoDePago = new MetodoPago();
    }

    public Integer getIdComprobante() {
        return idComprobante;
    }

    public void setIdComprobante(Integer idComprobante) {
        this.idComprobante = idComprobante;
    }

    public String getFechaEmision() {
        return fechaEmision;
    }

    public void setFechaEmision(String fechaEmision) {
        this.fechaEmision = fechaEmision;
    }

    public String getHoraEmision() {
        return horaEmision;
    }

    public void setHoraEmision(String horaEmision) {
        this.horaEmision = horaEmision;
    }

    public double getTotal() {
        return total;
    }

    public void setTotal(double total) {
        this.total = total;
    }

    public MetodoPago getMetodoDePago() {
        return metodoDePago;
    }

    public void setMetodoDePago(MetodoPago metodoDePago) {
        this.metodoDePago = metodoDePago;
    }

    @Override
    public String toString() {
        String resultado = "";
        resultado += "================\n";
        resultado += "Comprobante\n";
        resultado += "----------------\n";
        resultado += "idComprobante: " + idComprobante  + "\n";
        resultado += "fechaEmision: "  + fechaEmision   + "\n";
        resultado += "horaEmision: "   + horaEmision    + "\n";
        resultado += "total: "          + total           + "\n";
        resultado += "metodoDePago: "   + "\n"; 
        resultado += metodoDePago;    
        resultado += "================\n";
        return resultado;
    }

    
}
